api/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --output-file=requirements.txt --strip-extras pyproject.toml
```

Comment:
Was the `--strip-extras` option causing problems?

Reply:
I just opened another pull request because this one had the incorrect python version again, but yes I think the strip extras may have removed those from the requirements-dev file.